app/controllers/LoginController.scala

```diff
@@ -0,0 +1,42 @@
+package controllers
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import akka.stream.Materializer
+import helpers.Constants
+import models.LoginDetails
+import play.api.i18n.{I18nSupport, MessagesApi}
+import javax.inject.Inject
+import play.api.mvc.{Action, AnyContent, Controller}
+
+import scala.concurrent.Future
+
+class LoginController @Inject()
+(val messagesApi: MessagesApi, val materializer: Materializer) extends Controller
+  with I18nSupport {
+
+  def login: Action[AnyContent] = Action.async { implicit request =>
+    Future{
+      Ok(views.html.login(LoginDetails.loginForm))
+    }
+  }
+
+  def loginSubmit: Action[AnyContent] = Action.async { implicit request =>
+    var loginDetails = LoginDetails(Constants.emptyString.toString, Constants.emptyString.toString)
+
+    LoginDetails.loginForm.bindFromRequest.fold({
+      _ => {
+        Future(Redirect(routes.LoginController.login()).flashing(Constants.failed.toString ->
+          Constants.incorrectUsername.toString
+        ))
+      }
+    },
+      {
+        details =>
+          loginDetails = details
```

Comment:
just use `details` the `loginDetails` isn't really needed, should work all fine without it